docs/class-definitions.md

```diff
@@ -42,6 +42,28 @@ Number of milliseconds after a user loses connection or closes their browser win
 
 Default is 120,000ms (2 minutes).
 
+#### cursors
+
+Configure the cursors API.
+
+##### outgoingBatchTime
+
+The interval at which a batch of cursors position is published. This is multiplied by the number of members in the space reduced by 1. (i.e. outgoingBatchTime \* (membersCount - 1)).
+
+Default 100ms.
+
+##### incomingBatchTime
+
+The interval at which the listener for cursor positions is updated with a single position.
```

Comment:
Just for clarity on this one - is it a single position per user, or for all users? E.g. if there are 10 cursors moving, will it take 10ms to update all of them by 1 position?

Reply:
> if there are 10 cursors moving, will it take 10ms to update all of them by 1 position

That's correct